CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# How to use this repo
+
+Please [open Issues](https://github.com/sensu-plugins/sensu-plugins-feature-requests/issues) for anything on your mind that will help the community succeed at having **the best collection of monitoring plugins.**
+
+Common requests are:
+
+* Transfer ownership of your rad Sensu plugin to this organization ([example][1])
+* Offer an awesome idea that will help others make great plugins ([example][3])
+
+If you're ever not sure about how you can help, reach out on [our Community Slack](https://sensucommunityslack.herokuapp.com/)!
```

Comment:
I've been thinking about this. Two links every time we mention Slack is going to be messy since it's two different call to actions and someone has to think too much to know what to do. It feels like a fair assumption that if someone has already signed up they can access it or find the little "or sign in" link at the bottom.